src/PostgreSQLCopyHelper.NodaTime.Test/NodaTimeExtensionsTest.cs

```diff
@@ -0,0 +1,292 @@
+﻿// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NodaTime;
+using Npgsql;
+using NUnit.Framework;
+using PostgreSQLCopyHelper.NodaTime;
+using PostgreSQLCopyHelper.NodaTime.Test.Extensions;
+
+namespace PostgreSQLCopyHelper.NodaTime.Test
+{
+    [TestFixture]
+    public class NodaTimeExtensionsTest : TransactionalTestBase
+    {
+
+        public class AllNodaTypesEntity
+        {
+            public LocalTime LocalTime { get; set; }
+
+            public LocalDateTime LocalDateTime { get; set; }
+
+            public ZonedDateTime ZonedDateTime { get; set; }
+
+            public OffsetTime OffsetTime { get; set; }
+
+            public Instant Instant { get; set; }
+
+            public OffsetDateTime OffsetDateTime { get; set; }
+
+            public Period Period { get; set; }
+
+            public LocalDate LocalDate { get; set; }
+        }
+
+        [Test]
+        public void Test_Interval()
+        {
+            CreateTable("interval");
+
+
+            var begin = new LocalDateTime(2020, 1, 23, 0, 12);
+            var end = new LocalDateTime(2020, 12, 8, 12, 44);
+
+            var subject = new PostgreSQLCopyHelper<AllNodaTypesEntity>("sample", "nody_time_test")
```

Comment:
```suggestion
            var subject = new PostgreSQLCopyHelper<AllNodaTypesEntity>("sample", "noda_time_test")
```

Reply:
Also would it be worth making these constants? as to not maintain them in the tests?

Reply:
I know my "Copy and Paste"-Pattern is bad. But for this, I would rather duplicate it, than jumping back and forth to see the actual definition. This is just my feeling.